pkgs/development/python-modules/beets/default.nix

```diff
@@ -433,7 +434,14 @@ buildPythonPackage rec {
           testPaths = [ ];
         };
       };
-      base = lib.mapAttrs (_: a: { builtin = true; } // a) passthru.plugins.builtins;
+      base = {
+        beetcamp = {
+          enable = false;
+          propagatedBuildInputs = [ beetcamp ];
+          testPaths = [ ];
+        };
+      }
+      // lib.mapAttrs (_: a: { builtin = true; } // a) passthru.plugins.builtins;
```

Comment:
I'd not touch `beets` at all in this PR... Also, I don't understand this change - are you simply adding `beetcamp` to the plugins list?